Dockerfile

```diff
@@ -20,4 +20,4 @@ WORKDIR /opt/representer
 
```

Comment:
Note for the review : I don't know why we target "x86_64-unknown-linux-musl" especially don't wanted to change that because I didn't know why it was there but if there is explanation I would be happy to know them !

Reply:
I assume you're wondering about `linux-gnu` versus `linux-musl`. `linux-gnu` is more common, it relies on `glibc`, a popular implementation of the C standard library by GNU. The Alpine docker image we're using (because it is a very small base) uses `musl` as its standard library instead of `glibc`. Let me know if there's more you'd like me to explain.

Reply:
Yes I wasn't able to comment in upper lines, you assumed well 😆 . Ok can 't test with the Docker locally as i'm on M1 Apple I will try to make an other docker locally for me to test in the next days.

Reply:
I see. That's probably an issue with `x86_64` though, the CPU architecture. I guess it should work the same way if you change it to test locally. But you can't commit that, because our servers are definitely x86 based. You could try with `aarch64-unknown-linux-musl`.

Reply:
FYI. I believe you should be able to run x86 images on your mac in general, using the `--platform linux/amd64` flag. See https://forums.docker.com/t/run-x86-intel-and-arm-based-images-on-apple-silicon-m1-macs/117123 for example.

On my machine (M2) I run the normal exercism x86 docker images. These are my docker settings in case this helps:

<img width="693" alt="Screenshot 2023-09-10 at 00 16 56" src="https://github.com/exercism/rust-representer/assets/286476/42846fa3-f199-4442-bf40-1b95535a1091">

Reply:
Thanks for the solution, worked well. I fixed the docker that wasn't working then and it simplify it a bit. Closing this thread as the main purpose of it has been answered